pkgs/servers/blockbook/default.nix

```diff
@@ -0,0 +1,51 @@
+{ buildGoPackage
+, lib
+, fetchFromGitHub
+, rocksdb
+, bzip2
+, zlib
+, packr
+, snappy
+, pkg-config
+, zeromq
+, lz4
+}:
+
+buildGoPackage rec {
+  pname = "blockbook";
+  version = "0.3.1";
+
+  goPackagePath = "blockbook";
+
+  src = fetchFromGitHub {
+    owner = "trezor";
+    repo = "blockbook";
+    rev = "v${version}";
+    sha256 = "0qgd1f3b4vavw55mvpvwvlya39dx1c3kjsc7n46nn7kpc152jv1l";
+  };
+
+  goDeps = ./deps.nix;
+
+  propagatedBuildInputs = [ bzip2 zlib snappy zeromq lz4 ];
+
+  buildInputs = [ pkg-config packr ];
+
+  preBuild = ''
+    export CGO_CFLAGS="-I${rocksdb}/include"
+    export CGO_LDFLAGS="-L${rocksdb}/lib -lrocksdb -lstdc++ -lm -lz -lbz2 -lsnappy -llz4"
+    packr clean && packr
+  '';
+
+  postInstall = ''
+    rm $bin/bin/{scripts,templates,trezor-common}
```

Comment:
Why remove them?

Reply:
These are used for the upstream node setup on Debian, but not relevant for us.

Reply:
Please note, once we have the tool in nixpkgs, I plan on testing it and eventually writing a NixOS module for it.